client/v2/client.go

```diff
@@ -61,6 +70,9 @@ type Client interface {
 	// Query makes an InfluxDB Query on the database. This will fail if using
 	// the UDP client.
 	Query(q Query) (*Response, error)
+
+	// Close releases any resources a Client may be using.
+	Close() error
```

Comment:
The only thing about this is that we're adding a `Close()` without an `Open()`. If we had an `Open()` it would be there we could do the name resolution. I think symmetry is important in this interface.